src/helpers/add-pr-late-label.ts

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2022 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { LATE_REVIEW } from '../constants';
+import { HelperInputs } from '../types/generated';
+import { octokit } from '../octokit';
+import { RestEndpointMethodTypes } from '@octokit/rest';
+import { each } from "bluebird";
+
+type PullResponseType = RestEndpointMethodTypes["pulls"]["get"]["response"];
+type ListResponseType = RestEndpointMethodTypes["pulls"]["list"]["response"];
+
+export class AddPrLateReviewLabels extends HelperInputs {
+  owner = '';
+  repo = '';
+}
+
+export const addPrLateReviewLabels = async ({ owner, repo }: AddPrLateReviewLabels) => {
+  var page = 1;
+  var pull_requests = await getPRs(owner, repo, page) as ListResponseType;
+
+  while (pull_requests.data.length > 0 ) {
```

Comment:
Check out [this example](https://github.com/ExpediaGroup/github-helpers/blob/main/src/helpers/manage-merge-queue.ts#L90) of paginating over all open pull requests! I think we could extract that into a shared util function and use it here

Reply:
Would you like me to implement this, or leave it a note for a future improvement?

Reply:
I opened [this PR](https://github.com/tpoehlman/github-helpers/pull/1) to your fork